### ISSUE TITLE
Add rank and give karma to various interpretations.

### ISSUE THREAD

**Issue by recmo** [dex-ethics/open-eth#7]
"An ability to rank and give karma to various interpretations."

**Comment by recmo** [comment]
What are "various interpretations"?

@NellInc We can add ratings and/or up/downvoting to dillemmas and cases. Would this solve this issue?

**Comment by NellInc** [comment]
Yes, some content is likely to be more consistent, well-defined that others. There will certainly be some trolls out there too. There needs to be a way to downvote the profane into obscurity, and exalt the best. Upvotes karma seems like a good solution.
